runtime/domain.c

```diff
@@ -206,17 +206,7 @@ static struct {
   atomic_uintnat barrier;
 
   caml_domain_state* participating[Max_domains];
-} stw_request = {
-  ATOMIC_UINTNAT_INIT(0),
-  ATOMIC_UINTNAT_INIT(0),
-  NULL,
-  NULL,
-  NULL,
-  NULL,
-  0,
-  ATOMIC_UINTNAT_INIT(0),
-  { 0 },
-};
+} stw_request;
```

Comment:
I guess you're asking whether this change is correct.  I think it semantically correct, as the default initialization will put zeros and NULL pointers everywhere.  Stylistically, writing initializers explicitly  enables the compiler to warn about missing initializers when the type `stw_request` is extended with more fields, I believe.  You can preserve this by just replacing `ATOMIC_UINTNAT_INIT(0)` with `0` in the original code.

Reply:
Ah, thanks - warnings for new fields is the reason! I'll push the initializer back